device/iot-device-client/src/main/java/com/microsoft/azure/sdk/iot/device/FileUpload.java

```diff
@@ -46,17 +46,14 @@ FileUploadSasUriResponse getFileUploadSasUri(FileUploadSasUriRequest request) th
         return new FileUploadSasUriResponse(responseMessagePayload);
     }
 
-    @SuppressWarnings("UnusedReturnValue") // Public method
-    IotHubStatusCode sendNotification(FileUploadCompletionNotification fileUploadStatusParser) throws IOException
+    void sendNotification(FileUploadCompletionNotification fileUploadStatusParser) throws IOException
     {
         IotHubTransportMessage message = new IotHubTransportMessage(fileUploadStatusParser.toJson());
         message.setIotHubMethod(HttpsMethod.POST);
 
         ResponseMessage responseMessage = httpsTransportManager.sendFileUploadNotification(message);
 
         validateServiceStatusCode(responseMessage, "Failed to complete the file upload notification");
-
-        return responseMessage.getStatus();
```

Comment:
status code is used when validating status code on line 57, so no need to return this.